packages/@react-spectrum/s2/stories/TableView.stories.tsx

```diff
@@ -151,6 +152,94 @@ const DynamicTable = (args: any) => (
   </TableView>
 );
 
+
+const DynamicTableWithCustomMenus = (args: any) => (
+  <TableView aria-label="Dynamic table" {...args} styles={style({width: 320, height: 208})}>
+    <TableHeader columns={columns}>
+      {(column) => (
+        <Column
+          width={150}
+          minWidth={150}
+          isRowHeader={column.isRowHeader}
+          menu={
+            <>
+              <MenuSection>
+                <MenuItem onAction={action('filter')}><Filter /><Text slot="label">Filter</Text></MenuItem>
+              </MenuSection>
```

Comment:
The line above the menu section when it's the first one and there is no Header looks weird. I checked our Menu stories and they all have headers. Is this because a `MenuSection` is [wrapping](https://github.com/adobe/react-spectrum/pull/7617/files#diff-2e5e770823421d95998aca9927fff4ed25401c995f89a8612ab797ca7870ebd5R794) the `Collection` where the `MenuItem`s are inserted into the `TableHeader`'s column cell?
![image](https://github.com/user-attachments/assets/1bd32f71-bf2d-4c31-844e-a40da78026d4)